src/components/VisualItem/VisualItem.module.scss

```diff
@@ -0,0 +1,39 @@
+
+.overlay{
+    padding: 0 0 5% 0;
+}
+
+.itemWrapper{
+    display: flex;
+    flex-direction: column;
+
+    cursor: pointer;
+
+    .imageWrapper{
+        display: flex;
```

Comment:
This line seems to have no effect `justify-content: center;`
This line seems to have very little effect ` display: flex;`

Reply:
It has effect on image position. If you look close to image area there is a grey border. Without these two lines image is not at center.